src/sushipy/execute.py

```diff
@@ -45,10 +45,20 @@ class Execute:
     def translate(self, data: TranslateData):
         """translates string (multiple replace)"""
 
+        # modify args to only keep the second argument
+        if data.args:
+            args_list = data.args.split()
+            if len(args_list) >= 2:
+                args = args_list[1]
+            else:
+                args = ""
+        else:
+            args = ""
+

```

Comment:
This solution looks good, but maybe the easier way would be to use [this method](https://github.com/dev-sushi/sushi/pull/17#issuecomment-1465045779).

Reply:
Yes, I wasn't able to get that working out of the box. That may need a separate PR if we want to use positional and keyword arguments.

Reply:
> Yes, I wasn't able to get that working out of the box. That may need a separate PR if we want to use positional and keyword arguments.

It's fine for now. Just a future note.